examples/slider/css/slider-thermostat.css

```diff
@@ -0,0 +1,121 @@
+/* CSS Document */
+
+/* CSS Document */
+
+.slider-multithumb {
+  padding: 6px;
+  width: 550px;
+}
+
+.slider-thermostat-vertical,
+.slider-thermostat-text {
+  margin-top: 1em;
+  padding: 6px;
+  width: 12em;
+}
+
+.slider-thermostat-vertical .label,
+.slider-thermostat-text .label {
+  font-weight: bold;
+}
+
+.slider-thermostat-vertical.focus,
+.slider-thermostat-text.focus {
+  padding: 4px;
+  border: 2px solid #005a9c;
+  border-radius: 5px;
+}
+
+.slider-thermostat-vertical .value {
+  margin: 0.5em;
+  position: relative;
+  top: 20px;
+  height: 1.5em;
+}
+
+.slider-thermostat-vertical .input-value {
+  padding-left: 38px;
+}
+
+.slider-thermostat-vertical .input-value input {
+  width: 4em;
+}
+
+.slider-thermostat-vertical .slider-group .value {
+  font-size: 80%;
+  color: currentColor;
```

Comment:
```suggestion
  fill: currentColor;
```

Reply:
@carmacleod 
Thank you for finding this, changing CSS to use `fill` property.